azure-pipelines.yml

```diff
@@ -0,0 +1,12 @@
+# Gradle
+# Build your Java project and run tests with Gradle using a Gradle wrapper script.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/java
+
+pool:
+  vmImage: 'macOS 10.13'
+
+steps:
+- script: |
+    npm install -g appium@beta
+    ./gradlew clean build xcuiTest -x test -x signArchives
```

Comment:
What about using https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/gradle?view=vsts like below?

```
- task: Gradle@2
  inputs:
    gradleWrapperFile: 'gradlew' 
    tasks: 'clean build' 
    options: 'xcuiTest -x test -x signArchives'
```

According to https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-Readme.md, we probably can use the `gradle` task.

Reply:
@KazuCocoa Noted. Neat and clear.